pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = -l
+addopts = -l --ignore=scipy/_lib/array_api_compat
```

Comment:
Still needed when running tests with `dev.py` as the test discovery will get there.